buildSrc/src/main/groovy/com/uber/okbuck/core/dependency/ExternalDependency.groovy

```diff
@@ -23,6 +23,10 @@ class ExternalDependency extends VersionlessDependency {
         }
 
         this.depFile = depFile
+
+        if (artifactType == "jar") {
```

Comment:
When I was testing this, it included AARs. Those AARs had classifiers such as "debug" which seemed to break dependency resolution. This was just to keep it simple and not have to make any additional changes.